src/GraphQL/DataLoader/IDataLoader.cs

```diff
@@ -11,12 +11,12 @@ public interface IDataLoader
         /// <summary>
         /// Dispatch any pending operations
         /// </summary>
-        /// <param name="cancellationToken">Optional <seealso cref="CancellationToken"/> to pass to fetch delegate</param>
+        /// <param name="cancellationToken">Optional <seealso cref="CancellationToken"/> to pass to the fetch delegate</param>
```

Comment:
Please also change 
`Provides a method to dispatch pending a operation to load data.`
to
`Provides a method to dispatch a pending operation to load data.`

Reply:
Done